### DISCUSSION TITLE
Revisiting the Definition of the SCI and the use of marginal over average emission factors

### DISCUSSION THREAD

**Discussion by ciril-emaps** [Green-Software-Foundation/sci#337] [General]
As stated in the SCI Specifications, the purpose of the SCI is: 

> Software systems cause emissions through the hardware that they operate on, both through the energy that the physical hardware consumes and the emissions associated with manufacturing the hardware.

I would like to start a new discussion about how intensity factors are used in the definition of the SCI. Currently, the emissions factors used are marginal emission factors, however the question is whether marginal emission factors are the appropriate metric to use in order to account for emissions of electricity usage. 

I think it would be valuable to clarify, whether the SCI is supposed to be used to account for carbon emissions, or to assess the impact of interventions (e.g. load shifting).

**Comment by atg-abhishek** [comment]
@ciril-emaps the purpose of the SCI is first and foremost to trigger behaviour change, hence assessing the impact of interventions in your example would be what we would prioritize over just the accounting of emissions. That said, accounting is one step in the journey towards enabling that ultimate goal of behaviour change. 

Do you have more thoughts that you could share here for consideration on the appropriateness of the marginal vs. average emissions factors?

**Reply by ciril-emaps** [reply]
@atg-abhishek thanks a lot for your comment and clarification. As mentioned in the last session of the WG, I opened this discussion here to put this topic on the agenda of this week's session so that we can talk about this in more detail. We are planning to share our thoughts during the meeting, but please let me know if you would like me to share some of my thoughts before the meeting.

**Reply by atg-abhishek** [reply]
Really liked the presentation from @corradio on this that is linked below!

**Comment by corradio** [comment]
For reference, these are the slides that were presented:
https://docs.google.com/presentation/d/1g5ghRhm-WhHoc098Xvu_7cYAA6AbHAItmGWpNxvGvBI/edit#slide=id.g158aa0f95e7_0_0
For the curious, I also made a more in-depth presentation at a webinar which might be insightful:
https://www.youtube.com/watch?v=hD0Fo94OvNQ

**Comment by ciril-emaps** [comment]
There are currently two open action items: 

- [ ] Reach out to LCA experts and organise a sharing session
- [ ] Develop a suggestion for the SCI

As discussed in the session today, we have reached out to LCA experts to provide insights to the working group, however we are still waiting for responses. They are: 

- Thomas Gibon is a researcher at the Luxemburg Institute of Science and Technology in their Life Cycle Sustainability Analysis Group
- Anders Bjørn is a researcher at the Technical University of Denmark in their Department of Environmental and Resource Engineering

In case any member of the WG could propose other LCA experts from their network, please do not hesitate to inform us. 

Regarding the suggestion, we do think that any suggestion will benefit from expert input. Depending how quickly experts will be available for the sharing session, we will provide a suggestion before or after it has taken place.

**Reply by Henry-WattTime** [reply]
WG: Waiting for response from LCA experts. If have other recommended LCA experts, please provide potential introductions. 

Link from Pindy from last week: https://express.adobe.com/page/dey6WTCZ5JKPu/
Jens Malmodin might be a good LCA expert to review spec, Pindy is indirectly connected. Ciril to reach out to existing contacts again.

**Reply by Henry-WattTime** [reply]
Pindy reached out to contact, hasn't heard back yet, will reach out again.

**Reply by Henry-WattTime** [reply]
Pindy has appointment with researcher in the next few weeks (Feb 8).

**Reply by PindyBhullar** [reply]
Key point - to connect with researchers in the LCA to provide a background of their methodologies used.

**Comment by ciril-emaps** [comment]
Will not be able to attend today, however: 

Both LCA experts have answered and redirected me to one of their colleagues. Will have calls with them as soon as possible.

**Comment by Henry-WattTime** [comment]
WG:
Ciril - hard to find a LCA expert
-providing suggested edits

**Reply by Henry-WattTime** [reply]
https://www.electricitymaps.com/blog/how-leading-organisations-should-measure

**Reply by ciril-emaps** [reply]
Thanks a lot for posting this resource Henry. The above resource is somewhat out of date and we are currently reviewing it. I would therefore recommend looking into our latest educational resources: 

- A [blog post](https://www.electricitymaps.com/blog/marginal-vs-average-real-time-decision-making) on Marginal vs. Average Real-Time Decision-Making:
- A [webinar](https://www.youtube.com/watch?v=hD0Fo94OvNQ) on data-driven grid decarbonisation
- Our [carbon accounting guide](https://www.electricitymaps.com/guides/accounting-guide)

**Reply by Henry-WattTime** [reply]
I also found this guide very helpful: https://ghginstitute.org/2021/04/21/the-most-important-ghg-accounting-concept-you-may-not-have-heard-of-the-attributional-consequential-distinction/
